recipes-wpe/wpeframework/wpeframework-ocdm-widevine_git.bb

```diff
@@ -9,4 +9,12 @@ DEPENDS += " widevine"
 SRC_URI = "git://git@github.com/WebPlatformForEmbedded/OCDM-Widevine.git;protocol=https;branch=master"
 SRCREV = "052b138536bf209981a9360cc9ce1ae990a792ed"
 
-FILES_${PN} = "${datadir}/WPEFramework/OCDM/*.drm"
+do_install_append() {
+    install -m 755 -d ${D}/${sysconfdir}/WideVine
+    install -m 755 ${S}/tools/keybox/testkeybox.bin ${D}/${sysconfdir}/WideVine/keybox.bin
```

Comment:
How big is this? Is this always needed or only in case of a test/debug? Else we might make it conditional to avoid increasing the filesize of the image unnecessarily.

Reply:
This is 4.0K size file. This is required to work Widevine properly: [reference in BR](https://github.com/WebPlatformForEmbedded/buildroot/blob/ba1db982336ca92d95464a0d0933c9c1ff0c71cb/package/wpe/wpeframework-cdmi/wpeframework-cdmi-widevine/wpeframework-cdmi-widevine.mk#L15)